### ISSUE TITLE
chronometer shows as Offline

### ISSUE THREAD

**Issue by pheiduck** [pi-hole/pi-hole#4520]
### Versions

- Pi-hole:  5.8
- AdminLTE:  5.10
- FTL:  5.13

### Platform
Raspberry Pi OS 11
Raspberry Pi  3, Model B+

### Expected behavior

chronometer shows as **Active**

### Actual behavior / bug

chronometer shows as **Offline**

```
|¯¯¯(¯)_|¯|_  ___|¯|___        Core: v5.8
| ¯_/¯|_| ' \/ _ \ / -_)        Web: v5.10
|_| |_| |_||_\___/_\___|        FTL: v5.13
 ——————————————————————————————————————————————————————————
  Hostname: raspberrypi        (Raspberry Pi 3, Model B+)
...
   Pi-hole: Offline            (Blocking: 145849 sites)
...
```

### Steps to reproduce

Update Pihole to latest Versions
Reboot the System (doesn’t matter)

**Comment by mybasementcloud** [comment]
I am also seeing this behavior on Ubuntu 20.04, when running pihole -c, on three of three upgraded clients.  Admin web page shows same client as active and operational, pihole status indicates system is running and up as expected.

**Comment by pheiduck** [comment]
`pihole status` Show everything is running.

**Comment by yubiuser** [comment]
This is a bug. We forgot chronometer as we changed the behavior of `pihole status` in https://github.com/pi-hole/pi-hole/pull/4485

https://github.com/pi-hole/pi-hole/blob/2ff10fcd0a5464255d9c004541e6b23d7b42662f/advanced/Scripts/chronometer.sh#L360-L363

**Comment by yubiuser** [comment]
Please try if 

`pihole checkout core fix/chronometer`

fixes the issue for you. You can go back on track with `pihole checkout master` anytime

**Comment by mybasementcloud** [comment]
Hi, tried the provided fix and that corrects the issue.

**Comment by pheiduck** [comment]
> Hi, tried the provided fix and that corrects the issue.

Confirmed, Thanks a lot for the fix @yubiuser!

**Comment by pheiduck** [comment]
Fix is merged into 5.8.1
